content/en/blog/_posts/2017-02-00-Caas-The-Foundation-For-Next-Gen-Paas.md

```diff
@@ -3,6 +3,8 @@ title: " Containers as a Service, the foundation for next generation PaaS "
 date: 2017-02-21
 slug: caas-the-foundation-for-next-gen-paas
 url: /blog/2017/02/Caas-The-Foundation-For-Next-Gen-Paas
+author: >
+  [Brendan Burns](https://twitter.com/brendandburns) (Microsoft and Co-founder of Kubernetes)
 ---
 
 _Today’s post is by Brendan Burns, Partner Architect, at Microsoft & Kubernetes co-founder._  
```

Comment:
I'd drop or revise this note. We now just credit the person and optionally their organization.